sema/subyt/sources.py

```
@@ -88,7 +89,16 @@
         return mimetypes.guess_type(identifier)[0]  # type: ignore
 
     @staticmethod
-    def make_source(identifier: str | Path) -> Source:
+    def make_source(
+        identifier: str | Path, *, unique_pattern: str | None = None
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)